examples/cgra/ga_scripts/plot.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python plot_CDFs.py ga_output GA ../relative_placement_output/rp_data RLMap
```

Comment:
Does this need to be a script? 🙂 Or could it just be in a `README.md` file for this dir?